docs/concepts/controllers/statefulsets.md

```diff
@@ -0,0 +1,171 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+---
+
+{% capture overview %}
+**StatefulSets are a beta feature in 1.5. This feature replaces the 
+Pet Sets feature from 1.4. Users of Pet Sets are referred to the 1.5 
```

Comment:
Address the user directly.

"If you've been using PetSet in your cluster in version 1.4, see the [Upgrade Guide](/docs/tasks/stateful-set/upgrade-from-petsets-to-stateful-sets/) for further information on how to upgrade existing Pet Sets to StatefulSets."